src/DefaultTheme.js

```diff
@@ -7,41 +7,42 @@
  *
  */
 
-import cx                         from 'classnames/bind';
+import cx                                               from 'classnames/bind';
```

Comment:
Maybe to keep this spacing short? Now `import` and `from` are far away and all lines are exceeding the length of 80 😕